tests/functional/spec/features/view_submitted_response/view_submitted_response.js

```diff
@@ -1,20 +1,25 @@
 import AddressBlockPage from "../../../generated_pages/view_submitted_response/address.page.js";
 import NameBlockPage from "../../../generated_pages/view_submitted_response/name.page.js";
 import SubmitPage from "../../../generated_pages/view_submitted_response/submit.page.js";
+import ThankYouPage from "../../../base_pages/thank-you.page";
 import ViewSubmittedResponsePage from "../../../generated_pages/view_submitted_response/view-submitted-response.page.js";
 
 describe("View Submitted Response", () => {
   beforeEach("Load the questionnaire", () => {
     browser.openQuestionnaire("test_view_submitted_response.json");
-  });
-
-  it("Given a questionnaire has view submitted response enabled and has been completed, when on the view response page, then the summary is displayed correctly", () => {
     $(NameBlockPage.answer()).setValue("John Smith");
     $(NameBlockPage.submit()).click();
     $(AddressBlockPage.answer()).setValue("NP10 8XG");
     $(AddressBlockPage.submit()).click();
     $(SubmitPage.submit()).click();
-    browser.url("/submitted/view-response");
+    expect(browser.getUrl()).to.contain(ThankYouPage.pageName);
+    expect($(ThankYouPage.title()).getHTML()).to.contain("Thank you for completing the Test");
+    $(ThankYouPage.savePrintAnswersLink()).click();
+    expect(browser.getUrl()).to.contain(ViewSubmittedResponsePage.pageName);
+  });
+
+  it("Given a questionnaire has view submitted response enabled and has been completed, when on the view response page, then the summary is displayed correctly", () => {
```

Comment:
Might be slightly clearer and consistent with the test below.
```suggestion
  it("Given I have completed a questionnaire with view submitted response enabled, When I am on the view submitted response page within 45 minutes of submission, Then the summary is displayed correctly", () => {
```